Added October 2017 Bourbon Tasting data. Still needs image URLs

jasonsmithtx opened the pull request.

jasonsmithtx committed:
Added October 2017 Bourbon Tasting data. Still needs image URLs

kevinnayar commented on whiskey/public/data/whiskies.json:
```diff
@@ -3171,6 +3171,136 @@
+  
+      ],
+      "profiles":[
+        
```
Can you remove this line break?

kevinnayar commented on whiskey/public/data/whiskies.json:
```diff
@@ -3171,6 +3171,136 @@
+      "type": "Bourbon",
+      "url": "http://wildturkeybourbon.com/about/our-products/",
+      "description": "Caramel and mint on the nose. Followed by robust vanilla, toasted oak, and spices",
+      "ratings":[
```
Generally, I alphabetize these, but it's just a stylistic preference. Currently, the front-end doesn't alphabetize them on the modals, but I could add that.

kevinnayar commented on whiskey/public/data/whiskies.json:
```diff
@@ -3171,6 +3171,136 @@
+      ],
+      "average_rating":73
+    },
+   {
```
mis-aligned

kevinnayar commented on whiskey/public/data/whiskies.json:
```diff
@@ -3171,6 +3171,136 @@
-    }
-  ]
+    },
+  {
```
mis-aligned. Generally, just ensure that you've run the JSON through a validator like this one jsonformatter.curiousconcept.com, so it doesn't break the whole site.

kevinnayar reviewed (commented):
@jasonsmithtx I recommended a few stylistic changes.

jasonsmithtx committed:
Update whiskies.json

Validated json and added November while I was at it. Sending you Nov images via text and you have Oct images from your own photos
